### PR TITLE
feat: New github action to update aliases.json

### PR DESCRIPTION
**Pull request by ChaituVR**
### Changes
- Adds a new action to update aliases.json very similar to https://github.com/snapshot-labs/brovider/blob/master/.github/workflows/update-rpc.yml
- So that we don't have to wait for approvals

## How to test:
- In your local run `ts-node scripts/newAlias.ts thank.eth newspace.eth` 
- To test action you may have to use `act` package

### CODE REVIEW

File: scripts/newAlias.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const aliases = require('../spaces/aliases.json');
+const https = require('https');
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  const oldSpace = args[0];
+  const newSpace = args[1];
+  if (!aliases[oldSpace]) aliases[oldSpace] = newSpace
```
Comment (wa0x6e) [inline scripts/newAlias.ts:8]:
```suggestion
  if (aliases[oldSpace]) {
    console.error('Alias already exists for ' + oldSpace)
    process.exit(1);
  }
  
  aliases[oldSpace] = newSpace
```

Let's return an error and fail the whole workflow instead when data are not valid

File: scripts/newAlias.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const aliases = require('../spaces/aliases.json');
+const https = require('https');
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  const oldSpace = args[0];
+  const newSpace = args[1];
```
Comment (wa0x6e) [inline scripts/newAlias.ts:7]:
```suggestion
  const newSpace = args[1];
  
  if (!oldSpace || !newSpace.endsWith('.eth')) {
    console.error('Invalid arguments')
    process.exit(1);
  }
  
```

Let's also exit early when argument are not valid

File: scripts/newAlias.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  
+  if (!oldSpace || !newSpace.endsWith('.eth')) {
```
Comment (ChaituVR) [inline scripts/newAlias.ts:9]:
```suggestion
  if (!oldSpace || !newSpace || !oldSpace.endsWith('.eth') || !newSpace.endsWith('.eth')) {
```

Comment (wa0x6e) [reply scripts/newAlias.ts]: I think every items in `args` are already casted to `string`, and we can call `endsWith` directly on it without checking for existence first

File: scripts/newAlias.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+const aliases = require('../spaces/aliases.json');
+const https = require('https');
+
+const args = process.argv.slice(2);
+if (args.length === 2) {
+  const oldSpace = args[0];
+  const newSpace = args[1];
+  
+  if (!oldSpace || !newSpace || !oldSpace.endsWith('.eth') || !newSpace.endsWith('.eth')) {
```
Comment (ChaituVR) [inline scripts/newAlias.ts:9]:
```suggestion
  if (!oldSpace.endsWith('.eth') || !newSpace.endsWith('.eth')) {
```

**Review by wa0x6e — APPROVED** [review]
tAck

**Comment by wa0x6e** [comment]
addAlias would be a better filename IMO

**Comment by ChaituVR** [comment]
> addAlias would be a better filename IMO

leaving it as newAlias.ts with respect to workflow name

**Pull request merged**